server/routes/addPrisoner/prisonerAddedController.test.ts

```diff
@@ -37,7 +37,7 @@ describe('Prisoner added', () => {
         .expect(res => {
           const $ = cheerio.load(res.text)
           expect($('title').text()).toMatch(/^Prisoner added -/)
-          expect($('#navigation').length).toBe(0)
+          expect($('#navigation').length).toBe(1)
```

Comment:
unrelated - just making display of navigation consistent between add prisoner/visitor journeys (so a few of these in add prisoner journey files)